# Timeout::TimeoutError

Raised by Timeout.timeout when the block times out.

[Timeout::TimeoutError Reference](https://ruby-doc.org/stdlib-2.5.0/libdoc/timeout/rdoc/Timeout::TimeoutError.html)